src/ProjectTemplates/test/WebApiTemplateTest.cs

```diff
@@ -36,14 +36,24 @@ public ITestOutputHelper Output
     [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/28090", Queues = HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
     [InlineData("IndividualB2C", null)]
     [InlineData("IndividualB2C", new string[] { "--use-program-main" })]
+    [InlineData("IndividualB2C", new string[] { "--use-minimal-apis" })]
```

Comment:
nit: I see a lot of instances of these flags, might be worth considering moving these to constants now since we are changing/adding more of these now, i.e. UseProgramMain, UseMinimalApis

Reply:
For the string itself you mean, or the whole array?

Reply:
Yeah I was just suggesting we introduce a constant per hardcoded string, i.e. "--use-minimal-apis", more of a just in case these ever change type of thing, if its too painful, don't bother

Reply:
Well actually, I guess if you have a whole bunch of repeated command arrays, maybe that's a better form of redundancy elimination, I'm just basically asking if to consider if there's a way to do this where we don't have 50 copies of the same string literal, especially if they are all identical